internal/manager/config.go

```diff
@@ -185,16 +187,77 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	return flagSet
 }
 
+// CheckKongNonDefaultWorkSpace invokes :workspaces/ws-name, checks workspace existence
+func (c *Config) CheckKongNonDefaultWorkSpace(ctx context.Context, adminURL string, wsName string, client *kong.Client) (bool, error) {
```

Comment:
nit: consecutive variable types can have their type signature collapsed:

```suggestion
func (c *Config) CheckKongNonDefaultWorkSpace(ctx context.Context, adminURL, wsName string, client *kong.Client) (bool, error) {
```

Reply:
https://github.com/Kong/kubernetes-ingress-controller/pull/1758/files#diff-f45c581008bda413c341fdf775a5220e0193e4d268db67eae49f6eabba06d729R191